atc/integration/integration_suite_test.go

```diff
@@ -33,19 +32,12 @@ var (
 var _ = postgresrunner.GinkgoRunner(&postgresRunner)
 
 var _ = BeforeEach(func() {
-	cmd = &atccmd.RunCommand{}
-
-	// call parseArgs to populate flag defaults but ignore errors so that we can
```

Comment:
always nice seeing comments like this go away

Reply:
yessir. even though my PR adds a ton more comments for deprecated lines of code. hehe